src/audio/pipeline/pipeline-schedule.c

```diff
@@ -55,12 +55,38 @@ static enum task_state pipeline_task(void *arg)
 			return SOF_TASK_STATE_COMPLETED;
 	}
 
+	if (p->trigger.cmd >= 0) {
```

Comment:
can we add some inline comments in the new and existing code to follow the flow now.

Reply:
I actually thought, that very little of my work had as many comments as this PR :-D There is a comment  already here at the next line